docs/book/go_docs.md

```diff
@@ -0,0 +1,24 @@
+## GoDoc
+
```

Comment:
I feel the content in this file is more like notes to users. Maybe we can create a more organized structure.

Reply:
Changed this just to be links to the GoDoc